src/vabackend.c

```diff
@@ -626,9 +675,17 @@ static VAStatus nvQueryConfigAttributes(
         int i = 0;
         attrib_list[i].type = VAConfigAttribRTFormat;
         attrib_list[i].value = VA_RT_FORMAT_YUV420;
-        if (drv->supports16BitSurface && cfg->profile == VAProfileHEVCMain10) {
-            attrib_list[i].value |= VA_RT_FORMAT_YUV420_10;
-        }
+        if (drv->supports16BitSurface) { 
+            switch(cfg->profile) {
+            case VAProfileHEVCMain12:
+            case VAProfileVP9Profile2:
+                attrib_list[i].value |= VA_RT_FORMAT_YUV420_12;
+                // Fall-through
+            case VAProfileHEVCMain10:
+            case VAProfileAV1Profile0:
+                attrib_list[i].value |= VA_RT_FORMAT_YUV420_10;
+                break;
+        }}
```

Comment:
Can you split these out onto separate lines? Having them like that looks like a bug.